framework/wazuh/cluster/master.py

```diff
@@ -359,9 +359,9 @@ def update_file(name, data):
                     for file_path, file_data, file_time in cluster.unmerge_agent_info(data['merge_type'],
                                                                                       decompressed_files_path,
                                                                                       data['merge_name']):
+                        full_unmerged_name = common.ossec_path + file_path
```

Comment:
```suggestion
                        full_unmerged_name = os.path.join(common.ossec_path, file_path)
```

Reply:
Added in https://github.com/wazuh/wazuh/pull/2902/commits/6613363e0ee0650d947a557a856e5e231bd90a12